### PR TITLE
Add GPU miner

### PR DESCRIPTION
**Pull request by Gustav-Simonsson**
* Add 'opencl' build tag and check GO_OPENCL env variable

### CODE REVIEW

**Comment by robotally** [comment]
| Vote | Count | Reviewers |
| :---: | :---: | :---: |
| :+1: | 0 |  |
| :-1: | 0 |  |

_Updated: Wed Aug 19 14:49:24 UTC 2015_

**Comment by Gustav-Simonsson** [comment]
Looks it compiles OpenCL headers even when the build tag is not set. Hmm.

File: cmd/geth/main.go [hunk 1]
```diff
@@ -726,6 +746,30 @@ func makedag(ctx *cli.Context) {
 	}
 }
 
+func gpuinfo(ctx *cli.Context) {
+	//os.Getenv("GO_OPENCL")
```
Comment (obscuren) [inline cmd/geth/main.go]: Please remove debug comments

**Comment by Gustav-Simonsson** [comment]
Superseded by https://github.com/ethereum/go-ethereum/pull/1869

**Pull request closed without merging**